h/zos.h

```diff
@@ -1195,6 +1195,7 @@ int safVerify5(int options,
 #define SAF_AUTH_ATTR_UPDATE      0x04
 #define SAF_AUTH_ATTR_READ        0x02
 
+#define AUTH_WITHOUT_LOG  0x10
```

Comment:
@jordanfilteau1995 I think it's better to place this between `AUTH_DATASET_TAPE` and `AUTH_SUPERVISOR` and give it value 0x02. Otherwise it falls into the range of the "DATASET" options. Other than that the PR looks good to me.